build-tools/packages/build-infrastructure/api-report/build-infrastructure.api.md

```diff
@@ -6,6 +6,7 @@
 
 import type { Opaque } from 'type-fest';
 import type { PackageJson as PackageJson_2 } from 'type-fest';
+import * as semver from 'semver';
```

Comment:
Nit: JS code usually imports items using `camelCase`, because imports are treated more like variables. But in TypeScript, module imports are more like namespaces, so I think `PascalCase` is more the expectation. I think we currently have a mix throughout our code.

Reply:
That said, is there a reason not to just import the module's members directly instead of pulling in the namespace?